example/spec/helpers/Providers.js

```diff
@@ -100,10 +101,22 @@ const throwIfApiReturnFail = (apiResult) => {
   }
 };
 
+const providerExists = async (id) => {
```

Comment:
Good call. Thoughts on this implementation? [`1b4c0d6` (#2533)](https://github.com/nasa/cumulus/pull/2533/commits/1b4c0d688eaf72668651ab8caadd447a8fcb7144)

Reply:
I commented in the source as I was afraid it would be lost in the comments, resolving this in favor of that comment chain as the implementation looks good other than that comment.